src/components/molecules/RadioField/RadioField.tsx

```diff
@@ -138,9 +138,12 @@ const Input = styled.input<InputStatus & GroupingControlsProps>`
     `}
 `;
 
-export interface RadioField extends FieldProps, InputProps, GroupingControlsProps {}
+export interface RadioField extends FieldProps, InputProps, GroupingControlsProps {
+  groupLabel?: string;
+}
 
 const RadioField = ({
+  groupLabel = '',
```

Comment:
Maybe I'm missing something but `groupLabel` is the `name` of the radio group right?

Reply:
groupLabel allows us to be unique on when creating the id. We expect there to never be 2 labels the same.